doc/installation.rst

```diff
@@ -1,23 +1,48 @@
 Installation
 ============
 
+.. _requirements:
+
+Requirements
+------------
+The random forest used in SMAC depends on a c++11 compatible compiler
+and on SWIG (>3.0).
+
+To install both requirements system-wide on a linux system with apt, 
+please call:
+
+.. code-block:: bash
+
+    sudo apt-get install gcc swig
```

Comment:
From what I read [here](https://packages.ubuntu.com/xenial/build-essential), `build-essentials` installs the necessary things to install Debian-packages in general. That includes gcc. I don't know if it is required, but it kinda seems to be standard(?)...